snapcraft/parts/plugins/python_plugin.py

```diff
@@ -64,3 +65,33 @@ def _get_system_python_interpreter(self) -> Optional[str]:
             confinement,
         )
         return interpreter
+
+    @classmethod
+    def post_prime(cls, step_info: StepInfo) -> None:
+        """Perform Python-specific actions right before packing."""
+        base = step_info.project_base
+
+        if base != "core24":
```

Comment:
Since only core22 and newer snaps will use this code, what do you think about flipping this logic?

This is to help future me when `core26` get released.

```suggestion
        if base == "core22":
```


Also random thought - a future refactor could implement `__eq__()` and `__gt__()` for bases so we could do:
`if base >= core24:`

Reply:
Regarding the future refactor: Once snapcraft includes `craft-platforms`, you could do:

````python
if craft_platforms.snap.get_distro_base_from_core_base(base) >= craft_platforms.DistroBase("ubuntu", "24.04"):
```

Reply:
@mr-cal you are totally right. I struggled with this a bit because the code that calls this `post_prime()` method is on the LifecycleService, which means it doesn't get called for core22 and older - so the check could be seen as superfluous
Regardless you are correct about the future-proofing so I changed it to `if base in ("core20", "core22")` to act as a bit of documentation too